app/helpers/public_document_routes_helper.rb

```diff
@@ -144,6 +144,44 @@ def topical_event_about_pages_path(object, options = {})
     append_url_options("/government/topical-events/#{slug}/about", options)
   end
 
+  def world_location_path(object, options = {})
+    slug = case object
```

Comment:
Is it worth pulling this out into a method so that it can be used across these two methods?

Reply:
Good idea.  I've moved this into a separate method method.